dockers/README.md

```diff
@@ -14,10 +14,10 @@ or with specific arguments
 ```bash
 git clone <git-repository>
 docker image build \
-    -t pytorch-lightning:base-cuda-py3.9-pt1.10 \
-    -f dockers/base-cuda/Dockerfile \
+    -t pytorch-lightning:base-cuda-py3.9-pt1.11 \
+    -f dockers/base-rocm/Dockerfile \
```

Comment:
I would keep `base-cuda` since no users are expected to use `base-rocm` until PL supports ROCm.
```suggestion
    -f dockers/base-cuda/Dockerfile \
```

Reply:
Corrected, sorry about that